lsp-fsharp.el

```diff
@@ -77,6 +110,7 @@
                   :major-modes '(fsharp-mode)
                   :notification-handlers (ht ("fsharp/notifyCancel" #'ignore)
```

Comment:
not part of this review but you may take a look how do we handle the notifications in `lsp-rust` and `lsp-java` and implement the same logic here.

Reply:
Maybe I'm missing something really obvious, but they both look rather similar to what I'm doing here. Could you point me to the code that you are referencing?

Reply:
Here it is: https://github.com/emacs-lsp/lsp-mode/blob/master/lsp-rust.el#L296 (although I am not 100% sure that the FSharp notifications are progress notifications).